server/api/service_gc_safepoint.go

```diff
@@ -38,6 +38,7 @@ func newServiceGCSafepointHandler(svr *server.Server, rd *render.Render) *servic
 
 // ListServiceGCSafepoint is the response for list service GC safepoint.
 // NOTE: This type is exported by HTTP API. Please pay more attention when modifying it.
+// This type is in sync with `pd/client/http/types.go`.
```

Comment:
nit: Please unify `pd/client/http/types.go` in the ListServiceGCSafepoint comment and `client/http/types.go` in the ServiceSafePoint comment. :)